honeybee_3dm/grid.py

```diff
@@ -0,0 +1,47 @@
+"""Creating Honeybee grid objects from rhino3dm surfces"""
+
+import rhino3dm
+from honeybee.model import Model
+from honeybee_radiance.sensorgrid import SensorGrid
+from honeybee_radiance.writer import model_to_rad_folder
+
+from .room import to_room
+from .face import to_face
+from .tojson import to_json
+from .togeometry import brep2d_to_face3d
+
+
+def to_grid(path):
```

Comment:
I don't think this makes much sense to parse the file every single time for each object type. There should ideally be one time that you read the file and then use the loaded objects and the functions to translate them as needed.

Reply:
Complied